go.mod

```diff
@@ -24,10 +24,10 @@ require (
 	github.com/spf13/cobra v1.2.1
 	github.com/spf13/viper v1.9.0
 	github.com/stretchr/testify v1.7.1
-	golang.org/x/crypto v0.0.0-20220511200225-c6db032c6c88 // indirect
-	golang.org/x/net v0.0.0-20220624214902-1bab6f366d9e
-	golang.org/x/sys v0.0.0-20220520151302-bc2c85ada10a
-	golang.org/x/text v0.3.7 // indirect
+	golang.org/x/crypto v0.1.0 // indirect
```

Comment:
Do we need changes in this file to be checked in?

Reply:
Yes, [this](https://faun.pub/understanding-go-mod-and-go-sum-5fd7ec9bcc34) can be useful to read. I am reading it right now.

Reply:
Why I had to run `go mod tidy` is because I added a new dependency `golang.org/x/exp/slices`.